python/ray/actor.py

```diff
@@ -1817,3 +1914,42 @@ def exit_actor():
             f"{worker.mode}. Call this API inside an actor methods"
             "if you'd like to exit the actor gracefully."
         )
+
+
+def _handle_tensor_transport(dst_actor, args):
```

Comment:
Add type hints.

`args` is vague. Rename to something like `tensor_objects` or `tensor_refs`.

Reply:
Not all of them refer to a tensor. Some of them may not be references at all, and some may be references but point to non-tensor data.

Reply:
Maybe `task_args`?

Reply:
Updated https://github.com/ray-project/ray/pull/52938/commits/dd735db476e6c339aecdb58641077704476ec4c4